packages/invoices/src/InboundShipment/DetailView/Footer.tsx

```diff
@@ -9,6 +9,7 @@ import {
   useTranslation,
   useNotification,
   AppFooterPortal,
+  InvoiceNodeStatus,
```

Comment:
updates here for the 'status crumbs', which is essentially passing the 'right' array of statuses to the `StatusCrumb` component and constructing a 'right' status log of status:date pairs